### PR TITLE
feat: Ar 2615 add flag in clients table to marks zombie clients and ignore while sending a message 

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

[AR-2615]

### Solutions

display is a client is invalid only in debug mode

Needs releases with:

- [ ] https://github.com/wireapp/kalium/pull/1145

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

[AR-2615]: https://wearezeta.atlassian.net/browse/AR-2615?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1174/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5014716)(5014716)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5014716)(5014716)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt [hunk 1]
```diff
@@ -48,56 +50,64 @@ private fun DeviceItemContent(
...
-                    modifier = Modifier.shimmerPlaceholder(visible = placeholder),
-                    imageVector = ImageVector.vectorResource(id = R.drawable.ic_devices),
-                    contentDescription = stringResource(R.string.content_description_remove_devices_screen_device_item_icon)
+                    .padding(start = MaterialTheme.wireDimensions.removeDeviceItemPadding)
+                    .weight(1f),
+            ) { DeviceItemTexts(device, placeholder) }
+        }
+        val (buttonTopPadding, buttonEndPadding) = getMinTouchMargins(minSize = MaterialTheme.wireDimensions.buttonSmallMinSize)
+            .let {
+                // default button touch area [48x48] is higher than button size [40x32] so it will have margins, we have to subtract
```
Comment (Garzas) [inline app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt:73]: Can we move it inside WireSecondaryButton to control the touch area and size inside Button?

**Review by Garzas — APPROVED** [review]
LGTM 🎸 
Just one small question 🤔

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1174/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5033628)(5033628)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5033628)(5033628)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt (hunk 1 above)
Comment (MohamadJaara) [reply app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt:73]: this code is out of the scope of this PR

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1174/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5033877)(5033877)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5033877)(5033877)-dev-release.apk)

**Pull request merged**